### DISCUSSION TITLE
my arctic code vault code isnt there

### DISCUSSION THREAD

**Discussion by greydoubt** [community/community#105313] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

my arctic code vault code isnt there

**Comment by faridnec** [comment]
Hey do you mean by the badge? if so you perhaps can try checking
```plaintext
https://github.com/<your-username>?achievement=pull-shark&tab=achievements
```
if it is there you can click "unhide". Previously I got a similar problem with other badges
